aten/src/ATen/native/Convolution.cpp

```diff
@@ -595,7 +591,26 @@ at::Tensor _convolution(
   params.deterministic = deterministic;
   params.cudnn_enabled = cudnn_enabled;
 
-  check_shape_forward(input, weight, bias, params, input_is_mkldnn);
+  check_shape_forward(input, weight_sizes, bias, params, input_is_mkldnn);
+
+  if (input.size(0) == 0 || input.size(1) == 0) {    
```

Comment:
Empty channel should work as well? Is it tested?

Reply:
There's no reason why it should not, but in this case the model itself rather than the input is busted ;-), so probably the right thing is to remove channels check from here?

Reply:
I would agree that this is most likely a bad model 👍 
You can keep it out of this for now, and if people really want it, it will be super simple to add back and add to the test.

Reply:
Done.